### PR TITLE
[sampleapp] Add sample tests for WebStorage

### PR DESCRIPTION
**Pull request by kangxu**
Impacted tests(approved): new 1, update 0, delete 0
Unit test platform: Crosswalk Project for Android 15.43.354.0
Unit test result summary: pass 1, fail 0, block 0

BUG=https://crosswalk-project.org/jira/browse/XWALK-4271

### CODE REVIEW

File: misc/sampleapp-android-tests/testapp/WebStorage/index.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+Authors:
+        Xu, Kang <kangx.xu@intel.com>
+
+-->
+
+<meta charset="utf-8">
+<meta name="viewport" content="initial-scale=1.0, maximum-scale=1.0, width=device-width">
+<script src="js/jquery-2.1.3.min.js"></script>
```
Comment (zqzhang) [inline misc/sampleapp-android-tests/testapp/WebStorage/index.html]: Please don't involve so many jquery.js

Is `sampleapp-android-tests/webrunner/jquery-1.10.2.min.js` OK?

File: misc/sampleapp-android-tests/sampleapp/WebStorage.html [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+<title>WebStorage</title>
+<link rel="author" title="Intel" href="http://www.intel.com">
+<div>
+  <p>Test purpose:</p>
+  <ol>
+    <li>Install web app "webstorage.apk" and launch it</li>
+    <li>Input text in "sessionStorage" and "localStorage" area, then click button "Add Storage"</li>
+    <li>Input text in "sessionStorage" or "localStorage" area, then click button "Remove Storage"</li>
+    <li>Click button "Clear Storage"</li>
+    <li>Relaunch test app</li>
```
Comment (zqzhang) [inline misc/sampleapp-android-tests/sampleapp/WebStorage.html]: Does this need to relaunch sampleapp-android-tests?

File: misc/sampleapp-android-tests/testapp/WebStorage/index.html [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+Authors:
+        Xu, Kang <kangx.xu@intel.com>
+
+-->
+
+<meta charset="utf-8">
+<meta name="viewport" content="initial-scale=1.0, maximum-scale=1.0, width=device-width">
+<script src="js/jquery-1.10.2.min.js"></script>
```
Comment (zqzhang) [inline misc/sampleapp-android-tests/testapp/WebStorage/index.html]: ???

File: misc/sampleapp-android-tests/sampleapp/WebStorage.html [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+<link rel="author" title="Intel" href="http://www.intel.com">
+<div>
+  <p>Test purpose:</p>
+  <ol>
+    <li>Install web app "webstorage.apk" and launch it</li>
+    <li>Input text in "sessionStorage" and "localStorage" area, then click button "Add Storage"</li>
+    <li>Input text in "sessionStorage" or "localStorage" area, then click button "Remove Storage"</li>
+    <li>Click button "Clear Storage"</li>
+    <li>Relaunch web app "webstorage"</li>
+    <li>Input text in "sessionStorage" and "localStorage" area, then click button "Add Storage"</li>
```
Comment (zqzhang) [inline misc/sampleapp-android-tests/sampleapp/WebStorage.html]: The logic is different from https://crosswalk-project.org/jira/browse/XWALK-4260

~~~
Reproduce Test Steps
====================
Project setup:
ionic start test
cd test
ionic plugin add cordova-plugin-crosswalk-webview
ionic build android
JS test:
1. Add an item: localStorage.setItem('1','test')
2. Restart the app (remove from background)
3. Get an item: localStorage.getItem('1') //Gives null
4. If item was found then do localStorage.clear() and repeat again by incrementing the test name. Typically happens in max 1-3 trials.
~~~

**Comment by kangxu** [comment]
@zqzhang 
I have updated it following your comments.
As the description in https://crosswalk-project.org/jira/browse/XWALK-4260, we need to restart sub app "webstorage" after clear storage

**Comment by kangxu** [comment]
@zqzhang

**Comment by kangxu** [comment]
@zqzhang 
I have updated it following your comments

**Comment by zqzhang** [comment]
After talking with @Honry @ibelem , @wanghongjuan will add a WRT localStorage usecase. So I revert this PR.

**Pull request merged**